react.js

```diff
@@ -2,9 +2,13 @@ module.exports = {
   'extends': [
     './es6.js',
     'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:jsx-a11y/recommended',
```

Comment:
It's time

Reply:
Out of curiosity, do you know if `lp-components` passes these a11y recommendations?